Update host groups for Cloud 25.05 (#4489)
centreon/centreon-documentation: cloud/monitoring/groups.md

## Creating a host group

> Go to the **Configuration > Hosts > Host Groups** menu and click **Add**

Reviewer: ```suggestion
Go to **Configuration > Hosts > Host Groups** and click **Add**.
```

> * The **Name** and **Alias** defines the name and the alias of the host group.

Reviewer: ```suggestion
* Use the **Name** and **Alias** fields to define a clear, relevant name for the host group.
```

Reviewer: where will the alias appear?

> * The **Group members** list allows us to add hosts in the hostgroup.

Author: ```suggestion
* The **Group members** field allows us to add hosts in the hostgroup.
```

> * The **Group members** field allows us to add hosts in the hostgroup.

Reviewer: ```suggestion
* Use the **Group members** field to add hosts to the hostgroup.
```

> * The [**Resource access rules**](../administration/resource_access.md) allows us to grant the group access to the resources.

Reviewer: ```suggestion
* Use the [**Resource access rules**](../administration/resource_access.md) section to grant members of the group access to the resources defined in the rule.
```

> * The **Geographic coordinates** field defines geographical coordinates used by the Centreon MAP module to position the resource on a map.

Reviewer: ```suggestion
* The **Geographic coordinates** field is used by the Centreon MAP module to position the resource on a map.
```

Reviewer: add link

> * The **Icon** indicates the icon that will be displayed in the list of host groups.

Author: ```suggestion
* The selected **Icon** will be displayed in the list of host groups.
```

> * The **Comments** allows us to make comments about the group.

Reviewer: - Do not use "allows us to". "us" sounds odd. if you really need to use "allows", write "allows users to"
but in this case it is not necessary.
- There is no need to state that the comments fields is used to enter comments. The only useful thing you can say is probably "Comments entered here are for internal reference only and will only be visible on this page." or, more synthetically, "Notes added here are only visible on this page".

## Creating a host group

Go to **Configuration > Hosts > Host Groups** and click **Add**.

> * Use the **Name** field to define a clear, relevant name for the host group. The name can be used to find the host group with the research bar.

Author: ```suggestion
* Use the **Name** field to define a clear, relevant name for the host group. The name can be used in the **Resource status** search bar to display the group's hosts.
```

> * Use the **Name** field to define a clear, relevant name for the host group. The name can be used in the **Resource status** search bar after the syntax "host_group:" to display the group's hosts.

Reviewer: ```suggestion
* Use the **Name** field to define a clear, relevant name for the host group. The name can be used in the **Resource status** search bar using the syntax "host_group:" to display the group's hosts.
```

* Use the **Alias** field to better describe the host group. The alias is only visible in the host groups list.
* Use the **Group members** field to add hosts to the hostgroup.
* Use the [**Resource access rules**](../administration/resource_access.md) section to grant members of the group access to the resources defined in the rule.
* The **Geographic coordinates** field is used by the Centreon [MAP module](../graphical-views/introduction-map.md) to position the resource on a map.
* The selected **Icon** will be displayed in the list of host groups.

> * The **Comments** added here are only visible on this page.

Author: ```suggestion
* The **Comments** added here are only visible on this window.
```
